redirects.js

```diff
@@ -52,6 +52,44 @@ const setup = (app) => {
   conflates a language part ("zh") with a region ("europe") so we want to keep zh out of the manifest JSON */
```

Comment:
There's one more change for this file:

```js
  /* send auspice to the auspice docs (currently hosted as github pages) */
  app.route("/auspice")
    .get((req, res) => res.redirect('https://nextstrain.github.io/auspice/'));
```

Should now point to an appropriate section in docs.nextstrain.org :)

Reply:
I would send to the auspice subproject: https://docs.nextstrain.org/projects/auspice/en/migrate-docs/

Do you agree?

Reply:
Leaving a placeholder "Todo" here as this probably wants to be https://docs.nextstrain.org/projects/auspice/en/latest/ but this requries the merge of Auspice branch `migrate-docs`